save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/WelcomeView.kt

```diff
@@ -207,7 +207,7 @@ class WelcomeView : AbstractView<WelcomeProps, IndexViewState>(true) {
                 }
 
                 a(classes = "text-info text-gradient font-weight-bold ml-2 mr-2") {
-                    attrs.href = "/#"
+                    attrs.href = "/#/${props.userInfo?.name}/settings/email"
```

Comment:
why `e-mail` here?

Reply:
By default, a link to the e-mail setting is opened, as in **gitee**